### DISCUSSION TITLE
elitebook 8470p when in csrutil disable none of the apps that use microphone or camera ask for them

### DISCUSSION THREAD

**Discussion by alamboglia** [chris1111/HP-Probook-EliteBook-Package-Creator-OC#117] [Q&A]
Hopefull either Chris or anybody here can help with this. When I change the config.plist to csrutil enable everything works perfect, but when csrtutil disabled all the applications that need the permission for camera and microphone (whatsap, web apps that use the webcam like webcamtoys.com, recording apps like Pro Tools) wont ask them and theres no way to add them in system preferences, security and privacy and their recording and visual capabilities wont work.

Anyone knows how to fix this or a workaround?

**Comment by chris1111** [comment]
if you are on Monterey its normal because you have a snapshot disk with csrutil authenticated-root disable
there is no way to resolved this if you need workarond install Big Sur and use SIP enable  from OpenCore

**Reply by alamboglia** [reply]
I'm in Big Sur sorry i forgot to add that. Big sur 11.6.2

**Reply by alamboglia** [reply]
the thing is I need to leave it disabled so I can use your fix for sd cards. Or in this case I have to choose one or the other can't have the two?

**Comment by chris1111** [comment]
You can try to Notarized the kext
